lib/driver.js

```diff
@@ -44,6 +45,9 @@ const NO_PROXY = [
 
 class SelendroidDriver extends BaseDriver {
   constructor (opts = {}, shouldValidateCaps = true) {
+    // `shell` overwrites adb.shell, so remove
+    delete opts.shell;
```

Comment:
It's apparently got to do with repl support.

https://github.com/appium/appium/blob/1.5/lib/parser.js#L7